packages/react-components-pro/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@vaadin/react-components-pro",
+  "version": "24.4.0-alpha8",
+  "description": "React wrappers for commercial Vaadin components",
+  "type": "module",
+  "module": "index.js",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "github.com/vaadin/react-components"
+  },
+  "dependencies": {
+    "@lit/react": "^1.0.0",
+    "@vaadin/react-components": "24.4.0-alpha8",
```

Comment:
nit: we could sort this alphabetically (should be changed in the release anyway).

Reply:
Done